.kokoro/nightly/integration.cfg

```diff
@@ -21,6 +21,11 @@ env_vars: {
   value: "java-docs-samples-testing"
 }
 
+env_vars: {
+  key: "DATASTORE_PROJECT_ID"
+  value: "java-docs-samples-testing"
```

Comment:
Because `GOOGLE_CLOUD_PROJECT` is set to the same and is from our template (https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/java_library/.kokoro/nightly/integration.cfg), I'd like to keep it the same in this PR. We should revisit this and make the change in the template itself if we find it appropriate to switch.

Reply:
sure.